ledger/error/src/main/scala/com/daml/error/definitions/DamlError.scala

```diff
@@ -20,6 +21,9 @@ class DamlError(
   // Automatically log the error on generation
   loggingContext.logError(this, Map())
 
+  override def context: Map[String, String] =
+    super.context ++ extraContext.view.mapValues(_.toString)
```

Comment:
I am wondering if it would be better define `extraContext` as `Map[String, String]` and control what is being put there on the caller side, wdyt? or I miss something?